### PR TITLE
Improve UI messaging across pages in deploy board

### PR DESCRIPTION
**Pull request by gzpcho**
Why is this change necessary?
- Users struggle understanding certain parts of the Teletraan UI (see attached JIRA ticket)
- Tech debt violating DRY principle leads to inconsistencies across pages

How does this PR address the issue?
- It corrects the most pertinent visual inconsistencies
- It provides clarity on use cases like stage creation and other forms
- It adds additional context to better understand errors or requirements

What effects does this change have?
- Minor UI text changes for consistency across pages
- Changing "clone_from" field in stage creation form from text field to dropdown for better usability 
- Button link to Scaling Activities in environment landing page

Test Plan:
- Ensure modified anchor tags route correctly
- Attach screenshot of new stage creation form (with dropdown)
- Ensure scaling activities works correctly

JIRA Ticket(s): [CDP-6968]( https://jira.pinadmin.com/browse/CDP-6968)

### CODE REVIEW

File: deploy-board/deploy_board/templates/configs/capacity.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <li><a href="/envs">Environments</a></li>
     <li><a href="/env/{{ env.envName }}/{{ env.stageName }}">{{ env.envName }} ({{ env.stageName }})</a></li>
     <li><a href="/env/{{ env.envName }}/{{ env.stageName }}/config">General Config</a></li>
-    <li class="active"><a href="/env/{{ env.envName }}/{{ env.stageName }}/config/capacity">Capacity</a></li>
+    <li class="active">Capacity</li>
```
[the remark this answers is not in the document]
Comment (gzpcho) [reply deploy-board/deploy_board/templates/configs/capacity.html:10]: That breadcrumb item is the one this page is currently on, and to be consistent with other pages it should be "disabled" per say

File: deploy-board/deploy_board/templates/environs/new_stage_modal.tmpl [hunk 2]
```diff
@@ -29,8 +29,14 @@
                         <label for="after" class="control-label col-md-2">Clone from</label>
 
                         <div class="col-md-10">
-                            <input type="text" class="form-control" name="from_stage"
-                                   placeholder="existing stage to clone from.">
+                            <select class="form-control" name="from_stage">
```
Comment (tylerwowen) [inline deploy-board/deploy_board/templates/environs/new_stage_modal.tmpl:32]: NIce

**Review by tylerwowen — APPROVED** [review]

**Pull request merged**